tests/unit/test_asn1time.cpp

```diff
@@ -80,6 +80,12 @@ TEST_F(Asn1TimeTest, fromTimePointToTimePointIsIdentical)
     EXPECT_THAT(t.toTimePoint(), Eq(tp_2017_3_27__19_00_38));
 }
 
+TEST_F(Asn1TimeTest, fromTimeTToTimeTIsIdentical)
+{
+    auto t = Asn1Time::fromTimeT(tt_2017_3_27__19_00_38);
+    EXPECT_THAT(t.toTimeT(), Eq(tt_2017_3_27__19_00_38));
+}
+
```

Comment:
Please also add a test for the out of bounds cases.

Reply:
For time_point these tests already exist.
This is not as simple for time_t because the "out of bounds" cases depend on the platform. On the platform I'm using, time_t is a 128 bit integer (which makes its scope way larger than Asn1Time's).
Do you have an idea for a platform independent out of bounds test case?

Reply:
As discussed, I've added a static_assert in the test case that fails should time_t not be large enough.